docs/products/clickhouse/howto/connect-with-clickhouse-cli.rst

```diff
@@ -6,7 +6,7 @@ It's recommended to connect to a ClickHouse® cluster with the ClickHouse® clie
 Use the ClickHouse® client
 --------------------------
 
-To use the ClickHouse® client across different operating systems, we recommend utilizing `Docker <https://www.docker.com/>`_. You can get the latest image of the ClickHouse client directly from `the dedicated page in Docker hub <https://hub.docker.com/r/clickhouse/clickhouse-client>`_.
+To use the ClickHouse® client across different operating systems, we recommend utilizing `Docker <https://www.docker.com/>`_. You can get the latest image of the ClickHouse Server which contains the most recent ClickHouse Client directly from `the dedicated page in Docker hub <https://hub.docker.com/r/clickhouse/clickhouse-server>`_.
```

Comment:
the most recent ClickHouse Client => the most recent ClickHouse client

Reply:
either `the ClickHouse server` or, if proper name, `ClickHouse Server`

Reply:
have made changes to [clickhouse-client-docker](https://github.com/aiven/devportal/tree/clickhouse-client-docker) branch